apps/condo/pages/news/[id]/index.tsx

```diff
@@ -256,9 +255,12 @@ const NewsItemCard: React.FC = () => {
         } else if (newsItemSharing.status === 'error') {
             status = ErrorMessage
         } else if (newsItemSharing.status === 'success') {
+            dateToShow = get(newsItemSharing, 'updatedAt', null)
```

Comment:
Is it possible to update newsItemSharing after submission? 
If so, then there may be an incorrect `updatedAt`

Reply:
Maybe we should prohibit updating NewsItemSharing with the `success` status via api, so that such situations definitely don't happen?

Reply:
OK for me, but we should leave the ability to update it to support users